daemon/rpc_servers_test.go

```diff
@@ -0,0 +1,204 @@
+package daemon
+
+import (
+	"context"
+	"testing"
+
+	"github.com/NordSecurity/nordvpn-linux/config"
+	"github.com/NordSecurity/nordvpn-linux/core"
+	"github.com/NordSecurity/nordvpn-linux/daemon/pb"
+	"github.com/NordSecurity/nordvpn-linux/test/category"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestServers(t *testing.T) {
+	category.Set(t, category.Unit)
+
+	server1Hostname := "server1"
+	server1Country := "Germany"
+	server1CountryCode := "de"
+	server1City := "Berlin"
+
+	server2Hostname := "server2"
+	server2Country := "France"
+	server2CountryCode := "fr"
+	server2City := "Paris"
+
+	server3Hostname := "server3"
+	server3Country := "Lithuania"
+	server3CountryCode := "lt"
+	server3City := "Vilnius"
+
+	servers := core.Servers{
+		// Server 1
+		// Virtual/P2P/Germany/Berlin/Nordlynx/OpenVPN TCP Obfuscated
+		{
+			Hostname: server1Hostname,
+			Locations: core.Locations{
+				{
+					Country: core.Country{
+						Name: server1Country,
+						Code: server1CountryCode,
+						City: core.City{
+							Name: server1City,
+						},
+					}},
+			},
+			Specifications: []core.Specification{
+				{
+					Identifier: core.VirtualLocation,
+					Values: []struct {
+						Value string "json:\"value\""
+					}{
+						{Value: "True"},
+					},
+				},
+			},
+			Groups: core.Groups{
```

Comment:
for this group add also the standard group, to be closer to the response from the API.